packages/neutrino-preset-jest/src/index.js

```diff
@@ -65,6 +65,14 @@ module.exports = neutrino => {
       };
     });
 
+  neutrino.config.module
```

Comment:
It isn't known for sure whether a linting rule has been added, so this needs to be wrapped in a conditional.

```js
if (neutrino.config.rules.has('lint')) {
```